docs/src/main/sphinx/release/release-413.md

```diff
@@ -0,0 +1,55 @@
+# Release 413 (12 Apr 2023)
+
+## General
+
+* Improve performance of queries involving window operations or
+  [row pattern recognition](/sql/pattern-recognition-in-window) on small
+  partitions. ({issue}`16748`)
+* Improve performance of queries with `TopN` operations. ({issue}`16753`)
```

Comment:
TopN ... isnt really an operation .. what do we call this in other entries?

Reply:
In this case, it's queries involving `row_number()` and `rank()` window functions.